lib/Runtime/Base/FunctionBody.h

```diff
@@ -591,6 +591,9 @@ namespace Js
 #endif
     public:
         uint frameHeight;
+#if ENABLE_NATIVE_CODEGEN
+        bool nativeEntryPointProcessed;
+#endif
 
```

Comment:
That's what I had thought too. But, it has been used in places in the code without checking for ENABLE_NATIVE_CODEGEN. I didn't want to change all those places.